src/contracts/core/DelegationManager.sol

```diff
@@ -538,29 +531,31 @@ contract DelegationManager is
      * @param strategy The strategy to increase the delegated delegatedShares and the depositScalingFactor for
      * @param existingDepositShares The number of deposit shares the staker already has in the strategy.
      * @param addedShares The shares added to the staker in the StrategyManager/EigenPodManager
-     * @param maxMagnitude The current max magnitude of the operator for the strategy
+     * @param slashingFactor The current slashing factor for the staker/operator/strategy
      */
     function _increaseDelegation(
```

Comment:
This method name is a bit misleading now, since it's called anytime a staker's shares increase, regardless of being delegated

Reply:
Previously we (incorrectly) only increased when a staker was delegated

Reply:
i hear that.... but im not sure what a better name is. hard to capture both "update DSF" and "increase delegation" in one name

the simplest thing would be `updateShares` but i feel it's important to capture the delegation portion? idk. im gonna leave this for this PR, but im open to any changes you want to make?

Reply:
Ya, more of a callout than anything. I don’t have a better name so fine keeping as-is